### ISSUE TITLE
Outdated DST offset for Asia/Famagusta timezone

### ISSUE THREAD

**Issue by sofiafrocha** [manuelmhtr/countries-and-timezones#19]
(First, thank you for your work on this package!)

Currently, the `Asia/Famagusta` timezone has the following data:
```js
{
    name: "Asia/Famagusta",
    aliasOf: null,
    country: "CY",
    dstOffset: 120,
    dstOffsetStr: "+02:00",
    utcOffset: 120,
    utcOffsetStr: "+02:00"
}
```

Which indicates that the timezone does not observe DST.    
However, this is no longer the case, as now both [`Asia/Famagusta`](https://www.timeanddate.com/time/change/cyprus/famagusta) and [`Asia/Nicosia`](https://www.timeanddate.com/time/change/cyprus/nicosia) observe DST. So the timezones should return the same UTC (120) and DST (180) offsets.

The [ICANN mailling list](https://mm.icann.org/pipermail/tz/2017-October/025336.html) and [ICANN data (search for "# Cyprus")](https://data.iana.org/time-zones/releases/tzdata2019c.tar.gz) has more context around the changes and links for news reports related to the change.

**Comment by manuelmhtr** [comment]
Hello @sofiafrocha :) thanks for noticing this issue. I'll fix it as soon as possible.

**Comment by manuelmhtr** [comment]
Hi @sofiafrocha, the fix for "Asia/Famagusta" is now available in [v2.2.0](https://www.npmjs.com/package/countries-and-timezones/v/2.2.0). Along with other 17 timezones updated and 89 added (it now complies with the [IANA timezones database](https://www.iana.org/time-zones). Happy coding 😄

**Comment by sofiafrocha** [comment]
Awesome, thank you! 😀😀
